test/utils.js

```diff
@@ -226,6 +227,24 @@ export async function graphqlQueryV2(query, variables, remoteUser = null, jwtPay
   return graphqlQuery(query, variables, remoteUser, schemaV2, jwtPayload, headers);
 }
 
+/**
+ * This function allows to test queries and mutations against schema v2.
+ * @param {string} query - Queries and Mutations to serve against the type schema. Example: `query Expense($id: Int!) { Expense(id: $id) { description } }`
+ * @param {object} variables - Variables to use in the queries and mutations. Example: { id: 1 }
+ * @param {object} authorization - The user token to add to the context.
+ * @param {object} remoteUser - The user to add to the context. It is not required.
+ */
+export async function oAuthGraphqlQueryV2(
+  query,
+  variables,
+  authorization,
+  remoteUser = null,
+  jwtPayload = null,
+  headers = {},
+) {
+  return graphqlQuery(query, variables, remoteUser, schemaV2, jwtPayload, headers, authorization);
```

Comment:
Seems like this can be simplified

```suggestion
  userToken = null,
  jwtPayload = null,
  headers = {},
) {
  return graphqlQuery(query, variables, userToken.user, schemaV2, jwtPayload, headers, userToken);
```